### DISCUSSION TITLE
Can I force zgas to be equal to metallicity?

### DISCUSSION THREAD

**Discussion by junhopen** [mboquien/cigale#105] [Q&A]
I want to set multiple metallicity parameters in my model. After reading the literature, I found that the nebular module assumes that zgas is equal to metallicity.
How should I set the parameters to follow this assumption?

**Comment by mboquien** [comment]
Hello. This depends on the version. With modern versions the ionized gas metallicity is set separately from the star metallicity. You can find the metallicity parameter in each of the `nebular` and stellar population (usually `bc03`) modules to set them individually..

**Reply by junhopen** [reply]
So they are independent of each other. Maybe I can do multiple fits, each with a different metallicity value, and keep the metallicity equal to zgas.

**Reply by mboquien** [reply]
Yes, that is a possibility.
